src/api/spec/queries/workflow_runs_finder_spec.rb

```diff
@@ -3,13 +3,13 @@
 RSpec.describe WorkflowRunsFinder do
   let(:workflow_token) { create(:workflow_token) }
   # GitHub
-  let!(:workflow_run_github_push) { create(:workflow_run, :push, :succeeded, token: workflow_token) }
-  let!(:workflow_run_github_tag_push) { create(:workflow_run, :tag_push, :succeeded, token: workflow_token) }
+  let!(:workflow_run_github_push) { create(:workflow_run, :push, :succeeded, token: workflow_token, event_source_name: '94561cd') }
```

Comment:
Maybe better to add the test case when the filter is active (button search is clicked) but the input value is empty. It will fail (AKA the `with_event_source_name` needs to be adapted I guess) because the result I'd expect is to show them all, but it shows nothing instead.

An alternative is to prevent submitting the filter if the input value is empty.

Reply:
addressed in [Empty filter input displays all the results](https://github.com/openSUSE/open-build-service/pull/14730/commits/de042cbbb969fbe31e9483975b970fa626b6814f)